AP_HAL_ChibiOS - Added support for Aeromind6X (#31187)
ArduPilot/ardupilot: libraries/AP_HAL_ChibiOS/hwdef/Aeromind6X/README.md

- IO-Debug  

### Communication
- Ethernet  
- **Telemetry**: 3x Telemetry Ports  
- **CAN**: 2x UAVCAN  

### Actuator Outputs
- **Main Outputs**: 8 Channels  
- **Aux Outputs**: 8 Channels  

### RC Input
- SBUS / DSM / RSSI  
- PPM Input  

### Power Inputs
- 2x SMBUS (I2C)  
- 2x UAVCAN  
- SBUS In  
- PPM In  

### Other Ports
- 2x USB Ports  - 1 JST GH + 1 Type C
- 1x TF Card Slot (Supports MicroSD for flight logs)  

Pinout
======
![Pinout](Pinout.png)

UART Mapping
============

| SERIAL Port | UART Interface | Label / Function   | Flow Control |
|-------------|----------------|--------------------|---------------|
| SERIAL1     | UART7          | TELEM1             | Yes           |
| SERIAL2     | UART5          | TELEM2             | Yes           |
| SERIAL3     | UART1          | GPS1               | No            |
| SERIAL4     | UART8          | GPS2               | No            |
| SERIAL5     | USART2         | TELEM3             | Yes           |
| SERIAL6     | UART4          | UART4 (General Use)| No            |
| SERIAL7     | UART3          | FMU_DEBUG          | No            |

UART7, USART2 and UART5 have flow control pins.

RC input
========

> <!--This is the most difficult section and varies widely.-->

Author: Yes It is valid markdown. reviewed with vs code editor preview and also we have the same file used in our docs too

  - CC EAL6+ certified secure element  
  - 46 KB user memory (up to 104 KB customizable)  
  - AES / 3DES encryption

### Onboard Sensors
- Accelerometer / Gyroscope: 3x ICM-4568 (Rev. 8)  
- Magnetometer: PNI RM3100  
- Barometer: 2x ICP20100  

### Electrical
- Input Voltage: up to 6 V  
- USB Input: 4.75 V – 5.25 V  

### Mechanical
- Weight: 150 g  
- Dimensions: 94 × 46 × 38 mm  

---

## Interfaces

### Debug Ports
- FMU-Debug  
- IO-Debug  

### Communication Interfaces
- Ethernet  
- 3x Telemetry Ports (TELEM1, TELEM2, TELEM3)  
- 2x CAN (UAVCAN)  

### Actuator Outputs
- 8x Main Outputs (via IO Processor)  
- 8x Auxiliary Outputs (direct from FMU)  

### RC Input
- SBUS  
- PPM  
- DSM  
- RSSI  

### Power Inputs
- 2x SMBUS (I2C)  
- 2x UAVCAN  
- SBUS In  
- PPM In  

### Other Ports
- 2x USB Ports  
  - 1x USB Type-C  
  - 1x JST-GH  
- 1x TF Card Slot (MicroSD, for flight logs)  

---

## Pinout

![Pinout Diagram](Pinout.png)

---

## UART Mapping

| SERIAL Port | UART Interface | Label / Function     | Flow Control |
|-------------|----------------|----------------------|---------------|
| SERIAL1     | UART7          | TELEM1               | Yes           |

> | SERIAL2     | UART5          | TELEM2               | Yes           |

Reviewer: ```suggestion
| SERIAL2     | UART5          | TELEM2/MAVLink2| Yes           |
```

| SERIAL3     | UART1          | GPS1                 | No            |
| SERIAL4     | UART8          | GPS2                 | No            |

> | SERIAL5     | USART2         | TELEM3               | Yes           |

Reviewer: ```suggestion
| SERIAL5     | USART2         | TELEM3/MAVLink2| Yes           |
```

| SERIAL6     | UART4          | UART4 (General Use)  | No            |

> | SERIAL7     | UART3          | FMU_DEBUG            | No            |

Reviewer: ```suggestion
| SERIAL7     | USART3         | FMU_DEBUG            | No            |

All UARTS are DMA capable
```

> **Note**: UART7, USART2, and UART5 support hardware flow control.

---

## RC Input

> RC input is configured via parameters and processed by the IO Processor. Dedicated ports are available for SBUS, PPM, and DSM inputs.

Reviewer: Which pins are tied to the IOMCU RCin pin and which are tied to the PI5 pin on the FMU??? Need to be explicit here . Also should mention: that bi-directional protocols require use of a UART port,  see https://ardupilot.org/plane/docs/common-rc-systems.html

---

## PWM Outputs

Supports up to **16 PWM outputs**:

- **Main Outputs (M1–M8)**: via STM32F103 IO Processor  
- **Auxiliary Outputs (A1–A8)**: via STM32H753 FMU  

> 

Reviewer: All Outputs are PWM and DShot capable except outputs A7/A8, which  are only PWM capable. Outputs M1-8 are bi-directional DShot capable.

---

## Battery Monitoring

> Includes **two power monitor ports** using 6-pin connectors. Configuration depends on the type of power brick used.

Reviewer: ```suggestion
Includes **two I2C power monitor ports** using 6-pin connectors. Configuration is defaulted for the first monitor to be INA2XX I2C type power monitor.
```

---

## Compass

> The internal compass is provided by the **PNI RM3100** sensor.

Reviewer: ```suggestion
An internal compass is provided by a RM3100  sensor. Often this compass is disabled and an externally positioned  compass used to avoid power interference.
```

---

## Analog Ports

Includes two native ADC inputs:

> - ADC3V3  

Reviewer: ```suggestion
- ADC3V3  ArduPilot pin designation is "13"
```

> - ADC6V6  

Reviewer: ```suggestion
- ADC6V6  ArduPilot pin designation is "12"
```

Refer to the **Pinout Diagram** above for details.

---

> 

Reviewer: Add a GPIOS section here:
GPIO Label Mapping:
Label          GPIO Number
-------------- ------------
A(1)         50
A(2)         51
A(3)         52
A(4)         53
A(5)         54
A(6)         55
A(7)         56
A(8)         57
FMU_CAP1       58
NFC_GPIO       60
SPEKTRUM_PWR   73
ALARM          77
MAIN(1)        101
MAIN(2)        102
MAIN(3)        103
MAIN(4)        104
MAIN(5)        105
MAIN(6)        106
MAIN(7)        107
MAIN(8)        108
